### PR TITLE
leanin_query4

### PR DESCRIPTION
**Pull request by Raghavshalu**
Solved issue #7 added(Query-4) solution.

### CODE REVIEW

File: Problems/Query-4/Query-4.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+### Issue 7
+
+---
+
+### Problem 
+
+Write an SQL query to print the first three characters of FIRST_NAME from the Worker table.
+
+### Solution:
+
```
Comment (ananya0504) [inline Problems/Query-4/Query-4.md]: These lines are not required, please delete them. Thanks!

**Review by ananya0504 — CHANGES_REQUESTED** [review]
Please do the required changes @Raghavshalu

**Comment by Raghavshalu** [comment]
@ananya0504 done didi

**Review by ananya0504 — APPROVED** [review]
Okay looks good now, thanks!

**Pull request closed without merging**